cajita/src/Cajita_HypreStructuredSolver.hpp

```diff
@@ -120,6 +120,8 @@ class HypreStructuredSolver
         : _comm( layout.localGrid()->globalGrid().comm() )
         , _is_preconditioner( is_preconditioner )
     {
+        HYPRE_Init();
```

Comment:
Is it legal to call `HYPRE_Init()` multiple times in a program?

Reply:
It is legal to call `HYPRE_Init()` and then `HYPRE_Finalize()` followed by another call to `HYPRE_Init()` to create multiple different HYPRE solvers at discrete times. I am not sure if it is legal or if there is an issue with multiple calls to `HYPRE_Init()` before a call to `HYPRE_Finalize()`. I do not see any specific info on such a case in the documentation for hypre.